pkgs/development/python-modules/pymodbustcp/default.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "pyModbusTCP";
+  version = "0.3.0";
+  format = "setuptools";
```

The current convention is to build everything like pyprojects (still compatible with `setup.py`-type stuff)
```suggestion
  pyproject = true;
```

---

Building the package with

```
nix-build -E 'with import <nixpkgs> {}; python3Packages.callPackage ./default.nix {}'
```

doesn't work:

```
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...

Traceback (most recent call last):
  File "/nix/store/hv8xyk68b48ybj912m3a3kz904s0djas-python3.13-pyproject-hooks-1.2.0/lib/python3.13/site-packages/pyproject_hooks/_impl.py", line 402, in _call_hook
    raise BackendUnavailable(
    ...<4 lines>...
    )
pyproject_hooks._impl.BackendUnavailable: Cannot import 'setuptools.build_meta'

ERROR Backend 'setuptools.build_meta:__legacy__' is not available.
error: Cannot build '/nix/store/njhsxx6h57a4dz72x52nzvhn0arlrw18-python3.13-pyModbusTCP-0.3.0.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/lxl51drwiagg1s69n163dadc8yzdm52w-python3.13-pyModbusTCP-0.3.0-dist
         /nix/store/wcp33qb23brjffsri3sghabcdab622ws-python3.13-pyModbusTCP-0.3.0

```

---

OK, changed
Thank you for your help